Sprint-1/implement/sum.test.js

```diff
@@ -1,3 +1,11 @@
+function sum(elements) {
+  // Filter out non-numeric values and sum the rest
+  const numericElements = elements.filter(element => typeof element === 'number');
+  return numericElements.reduce((total, num) => total + num, 0);
+}
+
+module.exports = sum;
+
```

Comment:
Why not use `const sum = require("./sum.js");` to import the function?

Reply:
Thanks I have added that.